tests/FSharp.Compiler.ComponentTests/Conformance/DeclarationElements/LetBindings/UseBindings.fs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.ComponentTests.Conformance.DeclarationElements.LetBindings
+
+open Xunit
+open FSharp.Test.Utilities.Compiler
+open FSharp.Test.Utilities.Xunit.Attributes
+
+module UseBindings =
+
+    [<Theory; Directory(__SOURCE_DIRECTORY__ + "/../../../resources/tests/Conformance/DeclarationElements/LetBindings", Includes=[|"UseBindingDiscard01.fs"|])>]
+    let ``UseBindings - UseBindingDiscard01.fs - `` compilation =
+        compilation
+        |> asFsx
+        |> compile
+        |> shouldSucceed
+        |> ignore
+
+    [<Fact>]
+    let ``Dispose called for discarded value of use binding`` () =
+        Fsx """
+type private Disposable() =
+    [<DefaultValue>] static val mutable private disposed: bool
```

Comment:
It's a bit weird that you check that the constructor is called exactly once but the Dispose method being called more than once still succeeds :)